CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Update the style of the "Link a Sub-Event" button. [#1015](https://github.com/rokwire/events-manager/issues/1015)
 - Update the style of all checkboxes. [#1022](https://github.com/rokwire/events-manager/issues/1022)
 - Update the hover and focus style of all buttons. Refactored and cleaned unused styles. [#1048](https://github.com/rokwire/events-manager/issues/1048)
+- Update the style of reset button. [#1061](https://github.com/rokwire/events-manager/issues/1061)
```

Comment:
I think the "Reset" button in calendars and the "Add Contact" button in user-events page has been changed. Maybe mention this either in the issue or in the PR.